src/components/index.tsx

```diff
@@ -1,17 +1,42 @@
+import { darkTheme, lightTheme, Provider } from "@adobe/react-spectrum";
 import { GoogleOAuthProvider } from "@react-oauth/google";
+import { App } from "components/App";
+import { Luigi } from "components/Luigi";
+import { Stefano } from "components/Stefano";
 import { AuthenticationProvider } from "providers/AuthenticationProvider";
-import { SignatureProvider } from "providers/SignatureProvider";
-import React from "react";
+import { SignatureProvider, useSignatureContext } from "providers/SignatureProvider";
+import React, { FC } from "react";
 import * as ReactDOM from "react-dom";
+import { BrowserRouter, Navigate, Route, Routes } from "react-router-dom";
 
-import App from "./App/App";
+import { Layout } from "./Layout";
 
-const AppWithProviders = () => {
+const AppRouter: FC = () => {
+    const { isDarkMode } = useSignatureContext();
+
+    return (
+        <Provider theme={isDarkMode ? darkTheme : lightTheme} colorScheme="light" height="100%">
+            <Routes>
+                <Route path="/" element={<Layout />}>
+                    <Route path="/" element={<App />} />
+                    <Route path="luigi" element={<Luigi />} />
+                    <Route path="stefano" element={<Stefano />} />
```

Comment:
These are the nested paths, each one points to a specific component they will render, all of these will render _**inside**_ Layout since they are nested and Layout uses the `<Outlet />` tag

Reply:
Is the nested defined with white space indentation or through the overloaded use of the path value or both 🤔 💭  or is this wired due to the use of the Outlet component in the Layout component? 🚀

Reply:
It's nested since they are child component of the other route, check how the parent route is an open `<Route>` tag
<img width="510" alt="Screen Shot 2023-05-15 at 11 19 00 PM" src="https://github.com/evargast/signature-generator/assets/52969178/31752dcc-501f-41c9-95c3-b0938838e3aa">

Reply:
true its not a JSX call like the others 🤯  ty ty!